code/game/dna/dna_modifier.dm

```diff
@@ -589,6 +589,8 @@
 		var/lock_state = src.connected.locked
 		src.connected.locked = 1//lock it
 
+		nanomanager.update_uis(src) // show the splash while this works
```

Comment:
Asking me to HTML it over this is akin to Riri bitching that you need to rewrite mechs if you touch one single code line of them